index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width" />
+
+    <meta
+      name="description"
+      content="Author: Amedzro Emmanuel Elikplim, School: Microverse, Project: Portfolio"
+    />
+
+    <link rel="stylesheet" href="styles.css" />
+
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      rel="preload"
+      as="style"
+      onload="this.onload = null; this.rel = 'stylesheet'"
+      href="https://fonts.googleapis.com/css2?family=Raleway:wght@100;500;700&family=Roboto&display=swap"
+    />
+
+    <link
+      rel="preload"
+      as="style"
+      onload="this.onload = null; this.rel = 'stylesheet'"
+      href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+      integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm"
+      crossorigin="anonymous"
+    />
+
+    <title>Portfolio</title>
+  </head>
+
+  <body>
+    <header>
+      <nav class="toolbar">
+        <ul class="toolbar-items">
+          <li><h3 id="logo-text">John Doe</h3></li>
+          <li><i class="fa fa fa-bars bar-icon"></i></li>
+        </ul>
+      </nav>
+    </header>
```

Comment:
- [ ] kindly I would recommend you to follow the structure of the previous reviewer. Why this is because when building the mobile menu the navigation list needs to be hidden but not the logo. this will help your code to be more maintainable. Kindly remove the logo in the list and use flex to order them properly.
### Your solution 
```sh
    <header>
      <nav class="toolbar">
        <ul class="toolbar-items">
          <li><h3 id="logo-text">John Doe</h3></li>
          <li><i class="fa fa fa-bars bar-icon"></i></li>
        </ul>
      </nav>
    </header>
```
#### A better solution 
```sh
<header class="toolbar">
  <h3 id="logo-text">John Doe</h3>
  <nav>
    <i class="fa fa fa-bars bar-icon"></i>
   </nav>
</header>
```

Reply:
```
<header class="toolbar">
  <h3 id="logo-text">John Doe</h3>
  <nav>
    <i class="fa fa fa-bars bar-icon"></i>
   </nav>
</header>
```
Implemented the above code structure and added styling